assets/src/components/hooks/useAudio.tsx

```diff
@@ -15,16 +15,17 @@ import { useCallback, useRef } from 'react';
 export const useAudio = (src?: string) => {
   const audioRef = useRef<HTMLAudioElement>(null);
   const [isPlaying, setIsPlaying] = useState(false);
+
   const playAudio = useCallback((event?: any) => {
     event?.preventDefault && event?.preventDefault(); // Fixes https://eliterate.atlassian.net/browse/MER-1503
 
     const audio = audioRef.current;
     if (audio) {
       if (audio.paused) {
         audio.currentTime = 0;
-        setIsPlaying(true);
-        audio.currentTime = 0;
-        audio.play();
+        audio.play().then(() => setIsPlaying(true));
```

Comment:
Found a bug here, where if the audio url wasn't a valid audio file, and you tried to play it, it would get stuck in the playing state.